### PR TITLE
Text utilities amd Lists

### PR DESCRIPTION
**Pull request by vanshitamoolchandani**
Created Text Utilities and List
tu heading
tu colored
tu aligned
list ordered
list unordered
list inline
list stacked

### CODE REVIEW

File: component/Text-Utilities/index.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+    <!-- TEXT UTILITIES BODY -->
+
+
+    <div class="component-body">
+        <h2 class="component-heading">Text Utilities</h2>
+        <h5 class="component-content"></h5>
+
+
+            <h4 class="component-subheading">> Text </h4>
```
Comment (AshutoshM29) [inline component/Text-Utilities/index.html:62]: Instead of using ">" you can use bullets

File: component/Text-Utilities/styles.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+@import url('/styles/styles-global/root.css');
+@import url('/styles.css');
+
+.tu-container{
+    text-align: center;
+}
+.tu-container .component-subheading{
+    text-align: center;
+}
+
```
Comment (AshutoshM29) [inline component/Text-Utilities/styles.css]: Remove space like this

File: styles/styles-global/root.css [hunk 3]
```diff
@@ -49,10 +49,55 @@ body{
...
+}
+
+.iframe-container{
+    padding-top: 3rem;
+}
+
+.iframe{
+    width: 50rem;
+    height: 15rem;
+    margin-right:auto;
```
Comment (AshutoshM29) [inline styles/styles-global/root.css]: You can use margin directly here
like => margin: 0 auto;

**Review by AshutoshM29 — CHANGES_REQUESTED** [review]
Nice work!
Good to merge!!

File: styles/styles-global/root.css (hunk 3 above)
Comment (vanshitamoolchandani) [reply styles/styles-global/root.css]: Thank you for pointing it out.

File: component/Text-Utilities/index.html (hunk 1 above)
Comment (vanshitamoolchandani) [reply component/Text-Utilities/index.html:62]: will change it,

File: component/Avatars/index.html [hunk 4]
```diff
@@ -60,13 +60,13 @@ <h5 class="avatar-content">Avatars are images that users can set as their profil
...
-            <img class="img-rd-l" src="/images/avatar-img.png">
-            <img class="img-rd-m" src="/images/avatar-img.png">
-            <img class="img-rd-s" src="/images/avatar-img.png">
+        <h4 class="component-subheading">> Classic Avatar</h4>
+        <h5 class="component-content">Classic Avatars shows your profile picture in round shape. As shown...</h5>
+        <section class="avatar-container">
+            <img class="avatar-rd avatar-xl" src="/images/avatar-img.png">
+            <img class="avatar-rd avatar-l" src="/images/avatar-img.png">
+            <img class="avatar-rd avatar-m" src="/images/avatar-img.png">
+            <img class="avatar-rd avatar-s" src="/images/avatar-img.png">
```
Comment (gauravsinhaweb) [inline component/Avatars/index.html]: use alt inside img tag for accessibility.

File: component/Avatars/index.html [hunk 5]
```diff
@@ -86,13 +86,13 @@ <h5 class="avatar-section-content">Small (4 rem)</h5>
...
-            <img class="img-sq-l" src="/images/avatar-img.png">
-            <img class="img-sq-m" src="/images/avatar-img.png">
-            <img class="img-sq-s" src="/images/avatar-img.png">
+        <h4 class="component-subheading">> Squared Avatar</h4>
+        <h5 class="component-content">Squared Avatars shows your profile picture in Squared shape. As shown...</h5>
+        <section class="avatar-container">
+            <img class="avatar-sq avatar-xl" src="/images/avatar-img.png">
+            <img class="avatar-sq avatar-l" src="/images/avatar-img.png">
+            <img class="avatar-sq avatar-m" src="/images/avatar-img.png">
+            <img class="avatar-sq avatar-s" src="/images/avatar-img.png">
```
Comment (gauravsinhaweb) [inline component/Avatars/index.html]: alt tag

File: component/Badges/index.html [hunk 6]
```diff
@@ -52,24 +52,24 @@ <h4 class="nav-heading">Components</h4>
...
+        <h4 class="component-subheading">> Round Badge</h4>
+        <h5 class="component-content">Round Badge shows your profile picture and status in round shape. As shown...</h5>
+        <section class="avatar-container">
+            <div class="badge-container"><img class="avatar-rd avatar-m" src="/images/avatar-img.png"><span
                     class="badge-m badge-round bg-online bg-left"></span></div>
-            <div class="badge-container"><img class="img-rd-s" src="/images/avatar-img.png"><span
+            <div class="badge-container"><img class="avatar-rd avatar-s" src="/images/avatar-img.png"><span
                     class="badge-s badge-round bg-away bg-right"></span></div>
-            <div class="badge-container"><img class="img-rd-xs" src="/images/avatar-img.png"><span
+            <div class="badge-container"><img class="avatar-rd avatar-xs" src="/images/avatar-img.png"><span
```
Comment (gauravsinhaweb) [inline component/Badges/index.html]: alt tag, apply this for all.

**Review by gauravsinhaweb — CHANGES_REQUESTED** [review]

**Comment by gauravsinhaweb** [comment]
keep your commit in 100 lines or around it. It is hard to review large files.

File: component/Badges/index.html (hunk 6 above)
Comment (gauravsinhaweb) [reply component/Badges/index.html]: UI looks good!

**Pull request merged**